install.sh

```
@@ -893,28 +978,37 @@ configure_auth() {
     elif command_exists lsof; then
         if lsof -i:"$port" >/dev/null 2>&1; then
             return 1 # Port is in use
         else
             return 0 # Port is available
         fi
-    elif command_exists ss; then
-        if ss -lnt | grep -q ":$port "; then
+    # Finally try a direct connection with timeout
+    else
+        # Try to connect to the port, timeout after 1 second
+        if (echo > /dev/tcp/localhost/$port) >/dev/null 2>&1; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)